[2021-03-03 00:15:41 +0530] [62781] [INFO] Starting gunicorn 20.0.4
[2021-03-03 00:15:41 +0530] [62781] [INFO] Listening at: http://0.0.0.0:5000 (62781)
[2021-03-03 00:15:41 +0530] [62781] [INFO] Using worker: sync
[2021-03-03 00:15:41 +0530] [62785] [INFO] Booting worker with pid: 62785
[2021-03-03 00:15:41 +0530] [62786] [INFO] Booting worker with pid: 62786
[2021-03-03 00:17:45 +0530] [62855] [INFO] Starting gunicorn 20.0.4
[2021-03-03 00:17:45 +0530] [62855] [ERROR] Connection in use: ('0.0.0.0', 5000)
[2021-03-03 00:17:45 +0530] [62855] [ERROR] Retrying in 1 second.
[2021-03-03 00:17:46 +0530] [62855] [ERROR] Connection in use: ('0.0.0.0', 5000)
[2021-03-03 00:17:46 +0530] [62855] [ERROR] Retrying in 1 second.
[2021-03-03 00:17:47 +0530] [62855] [ERROR] Connection in use: ('0.0.0.0', 5000)
[2021-03-03 00:17:47 +0530] [62855] [ERROR] Retrying in 1 second.
[2021-03-03 00:17:48 +0530] [62855] [ERROR] Connection in use: ('0.0.0.0', 5000)
[2021-03-03 00:17:48 +0530] [62855] [ERROR] Retrying in 1 second.
[2021-03-03 00:17:49 +0530] [62855] [ERROR] Connection in use: ('0.0.0.0', 5000)
[2021-03-03 00:17:49 +0530] [62855] [ERROR] Retrying in 1 second.
[2021-03-03 00:17:50 +0530] [62855] [ERROR] Can't connect to ('0.0.0.0', 5000)
